eng/jacoco-test-coverage/pom.xml

```diff
@@ -149,7 +149,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-data-appconfiguration</artifactId>
-      <version>1.2.0-beta.1</version> <!-- {x-version-update;com.azure:azure-data-appconfiguration;current} -->
+      <version>1.1.7</version> <!-- {x-version-update;com.azure:azure-data-appconfiguration;current} -->
```

Comment:
Should we do a patch release from a release branch?

Reply:
App Config should be fine. Since it has no new feature implemented

Reply:
I think we want to introduce this pattern for patch releases so that we do not accidentally ship any "preview/in-progress" work.

Reply:
I see. I will create another one.